Misc/NEWS.d/next/Library/2021-10-08-04-11-55.bpo-10716.QSRVK2.rst

```diff
@@ -0,0 +1,3 @@
+Migrated pydoc to HTML5 (without changing the look of it). Side effet is to
```

Comment:
```suggestion
Migrated pydoc to HTML5 (without changing the look of it). Side effect is to
```

Reply:
My french is leaking in my english... fixed.